packages/cloud-cognitive/src/components/ComboButton/_combo-button.scss

```diff
@@ -11,55 +11,118 @@
 @import 'carbon-components/scss/components/overflow-menu/overflow-menu';
 
 @include exports($name: combo-button) {
-  /// Sets the prefix for all IBM Security-specific classes.
-  /// @access private
-  /// @type String
-  $security--prefix: security;
+  $blockClass: #{$pkg-prefix}--combo-button;
 
-  .#{$security--prefix}--combo-button {
+  .#{$blockClass} {
     position: relative;
     display: inline-flex;
   }
 
-  .#{$security--prefix}--combo-button__action {
+  .#{$blockClass}__action {
     @include text-overflow;
   }
 
-  .#{$security--prefix}--combo-button
-    .#{$security--prefix}--combo-button__overflow-menu {
-    width: carbon--mini-units($count: 6);
+  .#{$blockClass}__btn--sm.#{$carbon-prefix}--btn {
+    min-width: 8rem;
+  }
+
+  .#{$blockClass}__btn--md.#{$carbon-prefix}--btn {
+    min-width: 7.5rem;
+  }
+
+  .#{$blockClass}__btn--lg.#{$carbon-prefix}--btn {
+    min-width: 7rem;
+  }
+
+  .#{$blockClass}__btn--lg.#{$carbon-prefix}--btn--lg:not(
+      .#{$carbon-prefix}--btn--icon-only
+    ) {
+    min-height: 3rem;
+  }
+
+  .#{$blockClass} .#{$blockClass}__overflow-menu {
     height: auto;
     border-left: carbon--rem($px: 1px) solid $ui-03;
     background-color: $interactive-01;
   }
+  .#{$blockClass} .#{$blockClass}__overflow-menu:disabled,
+  .#{$blockClass} .#{$blockClass}__overflow-menu[disabled],
+  .#{$blockClass}__overflow-menu--disabled.#{$blockClass}__overflow-menu.#{$prefix}--overflow-menu:hover {
+    background-color: $disabled-02;
+    cursor: not-allowed;
+  }
+
+  .#{$blockClass}__overflow-menu__icon--disabled {
+    fill: $disabled-03;
+  }
+
+  .#{$blockClass}__overflow-menu--sm.#{$blockClass}__overflow-menu {
+    width: carbon--mini-units($count: 4);
+  }
+
+  .#{$blockClass}__overflow-menu--md.#{$blockClass}__overflow-menu {
+    width: carbon--mini-units($count: 5);
+  }
+
+  .#{$blockClass}__overflow-menu--lg.#{$blockClass}__overflow-menu {
+    width: carbon--mini-units($count: 6);
+  }
```

Comment:
Again, mini-units are getting removed. Let’s stick to spacing or rem here.

https://github.com/carbon-design-system/ibm-cloud-cognitive/pull/2130#discussion_r953059175